From bc840b69d51538f966d8d4800cb0ab460169971c Mon Sep 17 00:00:00 2001
From: Max Kellermann <max@duempel.org>
Date: Sun, 12 Oct 2014 08:38:52 +0200
Subject: [PATCH 3/3] Makefile.am: distribute systemd/mpd.socket

The file systemd/mpd.service.in is being distributed implicitly, but
systemd/mpd.socket is not and needs to be added to EXTRA_DIST.
---
 Makefile.am | 1 +
 2 files changed, 2 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 96c387f..2f3ab90 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2194,4 +2194,5 @@ EXTRA_DIST = $(doc_DATA) autogen.sh \
 	test/test_archive_zzip.sh \
 	$(wildcard scripts/*.sh) \
 	$(man_MANS) $(DOCBOOK_FILES) doc/mpdconf.example doc/doxygen.conf \
+	systemd/mpd.socket \
 	src/win32/mpd_win32_rc.rc.in src/win32/mpd.ico
-- 
2.1.2

